buildroot/share/PlatformIO/variants/MARLIN_F407VE/PeripheralPins.c

```diff
@@ -28,8 +30,8 @@
  *******************************************************************************
  * Automatically generated from STM32F407Z(E-G)Tx.xml
  */
-#include <Arduino.h>
-#include <PeripheralPins.h>
+#include "Arduino.h"
+#include "PeripheralPins.h"
```

Comment:
If these changes are not needed, we can leave them as "external header" style includes.

Reply:
these changes are there because I synced the files from ST Micro. the less you change them the better (easier to resync in future) actually they differed so much for the black that I had to work a lot on them to bring them up-to-date (same for the  #pragma once)

Reply:
there are some other changes which look "stupid" in those files (like the constants' change which don't really change value) but I did it anyway so we are in sync with them

Reply:
or the #if !defined..... please let's discuss about making changes before actually doing them....

Reply:
I think we can and should change the includes to use pointy braces where appropriate. It better expresses where we should expect to find those headers if we go looking for them.

Is it really necessary to arrange a tête-à-tête for innocuous style changes while I am studying the code? If you prefer that I leave them alone, of course that will just be … for now. I just can't abide `! defined (...)` when we have `#ifndef`.

Reply:
no...problem isn't in the "if ! defined" it's just in "file difference"
when you have to resync the files if the line has been modified you have to read it and understand what has been changed. if you massively replace something (let's say spaces) you force the one resyncing to read anyway the whole line to check if the only difference is in the spaces or in some significant content
the Periphalpins.c is an example: how can you tell if a register changed?
leaving the line "untouched" won't "pop up" in a "file compare" so you, reviewer or maintainer, WILL ignore it without even reading it. if any change pops up anyone thinks "what happened here?" wasting his time.

Reply:
...and that's why the black was so outdated: there were a lot of changed lines and instead of bothering reading them all someone said "let's just fix the stm to version 5.3..or 4... I don't remember" so leaving it "fixed" in time and making it even harder to resync as time/changes went on.